sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java

```diff
@@ -630,6 +633,30 @@ public void removeOnRPCNotificationListener(FunctionID notificationId, OnRPCNoti
 	public void start(){
 		if (proxy == null) {
 			try {
+				if(transport!= null  && transport.getTransportType() == TransportType.MULTIPLEX){
+					//Do the thing
+					MultiplexTransportConfig multiplexTransportConfig = (MultiplexTransportConfig)(transport);
+					final MultiplexTransportConfig.TransportListener devListener = multiplexTransportConfig.getTransportListener();
+					multiplexTransportConfig.setTransportListener(new MultiplexTransportConfig.TransportListener() {
+						@Override
+						public void onTransportEvent(List<TransportRecord> connectedTransports, boolean audioStreamTransportAvail, boolean videoStreamTransportAvail) {
+
+							//Pass to submanagers that need it
+							if(videoStreamingManager != null){
```

Comment:
This may get called before the videoStreamingManager is initialized. Because submanagers are initialized after proxy is connected. So `videoStreamingManager.handleTransportUpdated()` may not be called. And the state won't change to ready.